sdks/python/tox.ini

```diff
@@ -208,3 +208,13 @@ commands =
   coverage report --skip-covered
   # Generate report in xml format
   coverage xml
+
+[testenv:py36-interactive]
+extras = test,interactive
+commands =
+  python setup.py nosetests --where 'apache_beam/runners/interactive' {posargs}
```

Comment:
Is it possible that anything else will be broken in interactive environment?

Reply:
TL;DR: For simplicity, I'll just change to run all tests.

It's possible that after setting up a Py36 (or any other) environment with interactive dependencies, some dependencies would conflict or be incompatible. Like the rollback we had, in some test setup (could be totally different from the tox env we have in this repo), some execution path used a newer versioned IPython through transitive dependency causing package usage to fail if there is no try-except for relative import path.
To ensure at least all test environments work without conflict in dependencies, we can use the same `commands` used by `testenv:py36-gcp`. It's just now we rerun many tests simply because the result of `pip install ...` might be different.
But the test suites are executed in parallel, this just adds more workload to Jenkins without increasing sequenced task elapse, so it should be fine.